trompace/mutations/property.py

```diff
@@ -0,0 +1,101 @@
+# Generate GraphQL queries for mutations pertaining to software applications.
+
+from .templates import mutation_create, mutation_update, mutation_delete, mutation_link
+from . import StringConstant, bool_to_str
+
+
+CREATE_PROPERTY = '''
+CreateProperty(
+{parameters}
+) {{
+  identifier
+}}
+'''
+
+CREATE_PROPERTYVALUESPECIFICATION = '''
+CreatePropertyValueSpecification(
+{parameters}
+) {{
+  identifier
+}}
+'''
+
+ADD_CONTROLACTION_PROPERTYVALUESPECIFICATION  = '''
+AddActionInterfaceThingInterfacee(
+  from: {{identifier: "{identifier_1}"}}
+  to: {{identifier: "{identifier_2}"}}
+  field: object
+)
+{{
+    from {{
+        --typename
+  }}
+  to {{
+       --typename 
+       ... on PropertyValueSpecification{{  
+        identifier
+        name
+    }}
+  }}
+}}
+'''
+
+
+def mutation_create_property(title: str, name: str, description: str, rangeIncludes: list):
+    """Returns a mutation for creating a property
+    Arguments:
+        name: The name of the property
+        description: An account of the property
+        rangeIncludes: accepts an array of possible node types for this content reference and can be used by the Component developer to limit the type of nodes (content types) that can be selected.
+    Returns:
+        The string for the mutation for creating the property.
+
+    """
+    range_inc = []
+    for name in rangeIncludes:
+        range_inc.append(StringConstant(name))
+    args = {
+        "title": name,
+        "name": name,
+        "description": description,
+        "rangeIncludes" : StringConstant(str([StringConstant(x) for x in rangeIncludes]))
```

Comment:
This is from the template of the query present in the CE API. The argument is a list of node types but with the " marks of of a string.

Reply:
if that's the case, then maybe we should have another class (`ListConstant`?) so that we can specify this syntax in a more intuitive manner, instead of using `str` and `StringConstant` to emulate the behaviour